cli/commands/rewards/deploydistributor.ts

```diff
@@ -34,7 +35,7 @@ export default class DeployMerkleDistributor extends BaseCommand {
         gas: 1500000,
         gasPrice: '30000000000000'
     })
-    //@ts-ignore
+    // @ts-ignore - says there's no access to _address...but it is accessible
```

Comment:
clearly still getting the hang of typescript 💪

Reply:
I think `._address` is the private field, I think you should be able to access it "properly", via `.options.address` https://web3js.readthedocs.io/en/v1.2.11/web3-eth-contract.html#options

Reply:
not so, for whatever reason :/ comes up as undefined.
![Screen Shot 2021-01-08 at 10 58 08 AM](https://user-images.githubusercontent.com/5539720/104045026-f7961e00-51ab-11eb-97b4-91160a9d31be.png)

Reply:
also..the irony lol ---^

Reply:
ahh..missed `.options.`. fixing and resolving!